### PR TITLE
XOR-42 [FIX] Ensure Menu Settings opens after clicking bottom menu

### PR DESCRIPTION
**Pull request by anshukdutt**
### Product areas affected

Fliplet Widget menu manager, interface.js
### What does this PR do?

Menu Links opens after clicking Bottom menu and bottom menu icons

### JIRA ticket

[JIRA ticket](https://weboo.atlassian.net/browse/XOR-42)

### Result

[Screen Recordings](https://api.media.atlassian.com/file/74b0370f-3aab-47d3-aeb8-f914aee68957/artifact/video_1280.mp4/binary?client=435f5956-a841-4ea9-80ca-d94a2672bba6&collection=&max-age=2592000&token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI0MzVmNTk1Ni1hODQxLTRlYTktODBjYS1kOTRhMjY3MmJiYTYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOjljYTBmN2Y3LWUwYmQtNDhhMy1hYjM4LTJhZDY4MmQzNTYzYiI6WyJyZWFkIl0sInVybjpmaWxlc3RvcmU6ZmlsZToyZGNkMThkNy05MDJkLTQxNDYtODZjOC1mYWEzZjQ0NTllNWYiOlsicmVhZCJdLCJ1cm46ZmlsZXN0b3JlOmZpbGU6NzRiMDM3MGYtM2FhYi00N2QzLWFlYjgtZjkxNGFlZTY4OTU3IjpbInJlYWQiXSwidXJuOmZpbGVzdG9yZTpmaWxlOjg5ZmQ1ZjgzLWE5ODQtNGY5NC04NTVhLWIzZGJjOGU0OTJjNSI6WyJyZWFkIl0sInVybjpmaWxlc3RvcmU6ZmlsZTo5NTIwODgxNS1lZmJjLTQyYTItYmRjNy0zZDBiNWYxOGU0ODMiOlsicmVhZCJdfSwiZXhwIjoxNjUxOTkyOTY2LCJuYmYiOjE2NTE5OTIwMDZ9.LK2ep5vrloyllBi8v_rFVymDjlfnigVsyfUegJQCqHA)

### Checklist

None
### Testing instructions

None
### Deployment instructions

None
### Author concerns

None

### CODE REVIEW

**Review by anerib — APPROVED** [review]
LGTM

File: js/interface.js [hunk 1]
```diff
@@ -804,6 +804,12 @@
   attachObservers();
   // Load menu widgets on startup
   loadCustomMenuWidgets();
+
+  Fliplet.Studio.onMessage((eventData) => {
+    if (eventData && eventData.data && eventData.data.type === 'menu-tab-changed') {
+      $('.nav.nav-tabs li a[aria-controls="menu-manager"]').trigger('click');
```
Comment (tonytlwu) [inline js/interface.js:810]: See https://getbootstrap.com/docs/3.3/javascript/#tabs

image

This is how you use Bootstrap's API to select a tab by name. Please use the `href` target, not the ARIA accessibility attribute, and use the Bootstrap API.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Incorrect usage of Bootstrap API. See previous review comment.

**Comment by anshukdutt** [comment]
Hi @tonytlwu, 
As suggested $(`.nav.nav-tabs a[href='#${eventData.data.tab}']`).tab('show'), I used _.tab('show')_ , but its not working (Menu Links gets active but it keeps on loading)
On the other hand Using _.trigger('click')_ working fine. 
Can you suggest

**Comment by tonytlwu** [comment]
@anshukdutt We can stick with `.trigger('click')` for now. It looks like there's another instance in the same file that uses the same pattern.

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**